### ISSUE TITLE
include bugpoint & llvm-reduce?

### ISSUE THREAD

**Issue by mr-c** [emscripten-core/emsdk#901]
Hello, while chasing down https://github.com/emscripten-core/emscripten/issues/15149 I tried to follow the directions at https://llvm.org/docs/HowToSubmitABug.html#middle-end-optimization-bugs but could not continue as neither `bugpoint` nor `llvm-reduce` where not present in `emsdk/upstream/bin` (or anywhere else as far as I could see)

**Comment by sbc100** [comment]
@dschuff what do you think?

**Comment by dschuff** [comment]
Hm, that's a good question. Those tools are not useful to the vast majority of users (most users don't get quite so far along in helping reduce bugs; thank you!), and it would be nice not to bloat the installation packages. Maybe we could add them to the -asserts package (where they would be dynamically linked anyway) but not the LTO pacakge?

Meanwhile if you can tell me what platform and version you are running I can easily build you some compatible binaries.

**Comment by sbc100** [comment]
Yes I think perhaps including them in the non-LTO build is the way to go.

**Comment by sbc100** [comment]
Just going through some old issues.  Is there any interest in still doing this, or can we close?
